src/uwtools/config/atparse_to_jinja2.py

```diff
@@ -23,21 +22,15 @@ def convert(
     :param dry_run: Run in dry-run mode?
     """
 
-    def lines() -> Generator[str, Any, Any]:
-        with readable(input_file) as f_in:
-            for line in f_in.read().strip().split("\n"):
-                yield _replace(line.strip())
-
-    def write(f_out: IO) -> None:
-        for line in lines():
-            print(_replace(line.strip()), file=f_out)
-
+    with readable(input_file) as f:
+        lines = f.read().split("\n")
```

Comment:
I'm curious if `readlines()` doesn't work as nicely when `readable` returns from `stdin`.

Reply:
`readlines()` leaves a newline on each line (which `split()` removes), so those still have to be dealt with in edge cases, but overall it leads to a small simplifications, so: Will use, thanks!